src/components/Wizard/Wizard.tsx

```diff
@@ -119,61 +112,87 @@ export default function Wizard<T>({
+		textCancelButton: 'Terug',
+		textConfirmButton: 'Oke',
+		onCancel: () => {
+			setIsOpen(false);
+		},
+		onConfirm: () => {
+			terminate();
+		},
+	};
+
 	return (
-		<Modal id="dms-upload-wizard" open onClose={() => terminate()} closeOnBackdropClick={false}>
-			<Modal.TopBar hideCloseButton={false} onCloseButton={() => terminate()}>
-				<ModalTopBarStyle styleAs="h4" as="h2">
-					Bestand uploaden voor {name}
-				</ModalTopBarStyle>
-			</Modal.TopBar>
-			<>
-				<Modal.Content>
-					<ModalContentStyle>
-						<Route
-							exact
-							path={basePath}
-							render={() => (
-								<Step1
-									getHeaders={getHeaders}
-									getPostUrl={getPostUrl}
-									onFileRemove={handleFileRemove}
-									onFileSuccess={getFile}
-									storedFiles={!file ? [] : ([file] as FileUploadProps['storedFiles'])}
-									httpMethod={uploadHTTPMethod}
-								/>
-							)}
-						/>
-						<Route
-							path={appendPathSegment(basePath, 'step2')}
-							render={() => (
-								<MetadataForm
-									{...metadataForm}
-									onChange={(data, valid, errors) => {
-										dispatch(setMetadata(data));
-										setIsValidForm(valid);
-									}}
-								/>
-							)}
-						/>
-					</ModalContentStyle>
-				</Modal.Content>
-				<WizardFooter
-					cancel={{ visible: true, onClick: clickToCancel, dataTestId: 'cancel-wizard' }}
-					previous={{ visible: appendTrailingSlash(location.pathname) !== basePath, onClick: () => history.push(basePath), dataTestId: 'previous-button' }}
-					next={{
-						visible: !!(appendTrailingSlash(location.pathname) === basePath && file),
-						onClick: () => history.push(appendPathSegment(basePath, 'step2')),
-						disabled: !file,
-						dataTestId: 'next-button',
-					}}
-					save={{
-						visible: appendTrailingSlash(location.pathname) !== basePath,
-						disabled: !isValidForm,
-						onClick: handleSubmit,
-						dataTestId: 'save-button',
-					}}
-				/>
-			</>
-		</Modal>
+		<>
+			{isOpen && <ConfirmTermination backdropOpacity={1} />}
+			<Modal id="dms-upload-wizard" open onClose={() => terminate()} closeOnBackdropClick={false}>
+				<Modal.TopBar hideCloseButton={false} onCloseButton={() => terminate()}>
+					<ModalTopBarStyle styleAs="h4" as="h2">
+						Bestand uploaden voor {name}
+					</ModalTopBarStyle>
+				</Modal.TopBar>
+				<>
+					<Modal.Content>
+						<ModalContentStyle>
+							<Route
+								exact
+								path={basePath}
+								render={() => (
+									<Step1
+										getHeaders={getHeaders}
+										getPostUrl={getPostUrl}
+										onFileRemove={handleFileRemove}
+										onFileSuccess={getFile}
+										storedFiles={!file ? [] : ([file] as FileUploadProps['storedFiles'])}
+										httpMethod={uploadHTTPMethod}
+									/>
+								)}
+							/>
+							<Route
+								path={appendPathSegment(basePath, 'step2')}
+								render={() => (
+									<MetadataForm
+										{...metadataForm}
+										onChange={(data, valid, errors) => {
```

Comment:
Not your doing but `errors` is unused and thus should be removed

Reply:
And it's gone ;-)